MapboxCoreNavigation/RouteController.swift

```diff
@@ -263,6 +276,12 @@ open class RouteController: NSObject {
 
         checkForUpdates()
         checkForLocationUsageDescription()
+        
+        tunnelIntersectionManager = TunnelIntersectionManager()
+        tunnelIntersectionManager?.delegate = self
+        tunnelIntersectionManagerCompletionHandler = { enabled, _ in
+            self.tunnelIntersectionManager?.isAnimationEnabled = enabled
+        }
```

Comment:
It might be cleaner to wrap this in an `initTunnelManager` function.

Reply:
Sure